pkg/ui/v1alpha3/backend.go

```diff
@@ -443,3 +423,27 @@ func (k *KatibUIHandler) AddEditDeleteTemplate(w http.ResponseWriter, r *http.Re
 	}
 	w.Write(response)
 }
+
+func (k *KatibUIHandler) getExperimentList(namespace string) ([]JobView, error) {
+	jobs := make([]JobView, 0)
+
+	el, err := k.katibClient.GetExperimentList(namespace)
+	if err != nil {
+		log.Printf("GetExperimentList for NAS failed: %v", err)
```

Comment:
it seems the experiment  list will include all kinds of experiment, not only NAS